### PR TITLE
Fixed wrong grammar and spelling, made grammar within list more consistent

### PR DESCRIPTION
**Pull request by claraphyll**
["Email" is a coating for metal surfaces.](https://www.duden.de/rechtschreibung/Email) An "E-Mail" is an electronic letter. Also, there are no blanks within compound words. 

Made the list of standards consistently not use passive forms.

### CODE REVIEW

**Comment by goldibex** [comment]
Ich bin kein Muttersprachler, aber für mich sieht es gut aus! @agnoster schau mal an?

File: content/version/1/4/code-of-conduct.de.md [hunk 1]
```diff
@@ -39,7 +39,7 @@ Dieser Verhaltenskodex gilt sowohl innerhalb des Projektbereichs als auch in öf
 
 ## Umsetzung
 
-Fälle von missbräuchlichem, belästigendem oder anderweitig nicht akzeptablem Verhalten können dem Projektteam unter [EMAIL ADRESSE EINFÜGEN] gemeldet werden. Alle Beschwerden werden geprüft und untersucht und werden zu einer Antwort führen, die angesichts der Umstände für notwendig und angemessen gehalten wird. Das Projektteam ist verpflichtet, über diejenigen, die Vorfälle gemeldet haben, Verschwiegenheit zu wahren. Weitere Einzelheiten zu speziellen Umsetzungsgrundsätzen können gesondert mitgeteilt werden.
+Fälle von missbräuchlichem, belästigendem oder anderweitig nicht akzeptablem Verhalten können dem Projektteam unter [E-MAIL-ADRESSE EINFÜGEN] gemeldet werden. Alle Beschwerden werden geprüft und untersucht und werden zu einer Antwort führen, die angesichts der Umstände für notwendig und angemessen gehalten wird. Das Projektteam ist verpflichtet, über diejenigen, die Vorfälle gemeldet haben, Verschwiegenheit zu wahren. Weitere Einzelheiten zu speziellen Umsetzungsgrundsätzen können gesondert mitgeteilt werden.
```
Comment (Dahie) [inline content/version/1/4/code-of-conduct.de.md:42]: "Alle Beschwerden werden geprüft und untersucht"
 "prüfen" und "untersuchen" is almost the same and could be reduced to: "Alle Beschwerden werden geprüft und werden zu einer Antwort führen, die angesichts der Umstände für notwendig und angemessen gehalten wird"

**Review by dsturm — APPROVED** [review]

**Comment by heunland** [comment]
Lgtm (native German speaker)

File: content/version/1/4/code-of-conduct.de.md (hunk 1 above)
Comment (d4l-w4r) [reply content/version/1/4/code-of-conduct.de.md:42]: "Alle Beschwerden werden geprüft und untersucht" is closer to the literal english version though. 
See "All complaints will be reviewed and investigated [...]"

While I prefer the shorter version as well, I'm not sure how literal the translations should stay

File: content/version/1/4/code-of-conduct.de.md [hunk 2]
```diff
@@ -14,16 +14,16 @@ Im Interesse der Förderung eines offenen und einladenden Umfeldes wollen wir un
 Beispiele für Verhaltensweisen, welche dazu beitragen ein positives Umfeld zu erzeugen, beinhalten:
 
 * Die Verwendung von einladenden und einbindenden Formulierungen
-* Abweichenden Ansichten und Erfahrungen wird Respekt entgegengebracht
-* Konstruktive Kritik wird höflich entgegengenommen
-* Fokussierung auf das, was das Beste ist für die Gemeinschaft
-* Verständnis zeigen gegenüber anderen Mitgliedern der Gemeinschaft
+* Respektieren von abweichenden Ansichten und Erfahrungen
+* Höfliches Entgegennehmen konstruktiver Kritik
```
Comment (d4l-w4r) [inline content/version/1/4/code-of-conduct.de.md:18]: Closer to the original english translation than before :+1:

File: content/version/1/4/code-of-conduct.de.md [hunk 3]
```diff
@@ -14,16 +14,16 @@ Im Interesse der Förderung eines offenen und einladenden Umfeldes wollen wir un
...
+* Fokussierung auf das Beste für die Gemeinschaft
```
Comment (d4l-w4r) [inline content/version/1/4/code-of-conduct.de.md:19]: Maybe something like "Fokussierung auf das Wohl der Gemeinschaft" ?
While the above is a very literal translation it kind of seems...off to my ears

**Review by d4l-w4r — APPROVED** [review]
Looks good to me! Stays very close to the original english version.
Just found a little thing that sounded a bit off to me...but that might be personal preference

**Pull request closed without merging**